make/stanford.make

```diff
@@ -254,12 +257,12 @@ projects[stanford_bean_types][subdir] = "stanford"
 projects[stanford_bean_types][type] = "module"
 projects[stanford_bean_types][download][type] = "git"
 projects[stanford_bean_types][download][url] = "git@github.com:SU-SWS/stanford_bean_types.git"
-projects[stanford_bean_types][download][tag] = "7.x-3.0"
+projects[stanford_bean_types][download][tag] = "7.x-3.2-alpha2"
```

Comment:
Ah, this was to match the deployer, which is at: https://github.com/SU-SWS/stanford-jumpstart-deployer/blob/7.x-5.x/production/product/jumpstart-academic/stanford.make#L17.  I put the deployer at 3.2-alpha2 because the product tests pull from the latest tests, which assume the latest module version.  But this repository is testing with linky_clicky/sites/uat, which have not been changed to require the latest module version.